org.jcryptool.visual.sphincs/src/org/jcryptool/visual/sphincs/files/Converter.java

```diff
@@ -76,7 +76,7 @@ public static String _byteToString(byte[] input) throws UnsupportedEncodingExcep
 		for (int i = 0; i < l; i++) {
 			output[i] = _hexStringToByte(input.substring(i * s, s + i * s));
 		}
-
+		new String("asdfasdf bytes");
```

Comment:
What is this

Reply:
Oops, seems like an artifact, thanks for spotting.